.idea/gradle.xml

```diff
@@ -3,17 +3,15 @@
   <component name="GradleSettings">
     <option name="linkedExternalProjectsSettings">
       <GradleProjectSettings>
-        <option name="disableWrapperSourceDistributionNotification" value="true" />
         <option name="distributionType" value="DEFAULT_WRAPPED" />
         <option name="externalProjectPath" value="$PROJECT_DIR$/code_samples/action_basics" />
         <option name="gradleHome" value="C:/Program Files (x86)/Gradle/gradle-2.9" />
-        <option name="gradleJvm" value="1.8" />
+        <option name="gradleJvm" value="1.8 202b1483.58" />
```

Comment:
can we leave this at more generic 1.8?

Reply:
Sure. Realistically, it doesn't matter because who knows what developers call SDKs in their projects. But I get your point. There's nothing special about this SDK. It's just 1.8.